### DISCUSSION TITLE
Script mLITE

### DISCUSSION THREAD

**Discussion by adydevrsian** [basoro/Khanza-Lite#167] [Q&A]
halo dok., saya ingin membedah aplikasi mlite ini dari awal akan tetapi ada kendala dengan alur yang ada dari index.php dan seterusnya.,
jadi untuk pemula seperti saya di sarankan untuk belajar apa terlebih dahulu untuk bisa menguasai aplikasi mlite ini.?

**Comment by basoro** [comment]
Bisa lihat di User Manual mLITE v.1

Ada di grup Telegram
